### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by JohnPMurray**
Test coverage improved for backend queries and resolvers.

2 tests don't pass 
  students accessing their own portfolio in portfolio by period
  remove judge from a portfolio period they are not assigned to

They shouldn't be hard to fix, but they also don't break anything so maybe we just leave them

### CODE REVIEW

File: backend/resolvers/queries/portfolioQuery.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ export function portfolioByPeriod(_, args, req) {
   if (args.studentUsername && req.auth.type !== ADMIN) {
     throw new UserError("Permission Denied");
   }
-  const { periodId } = args;
+  const periodId = args.periodId;
```
Comment (johnbyrne727rit) [inline backend/resolvers/queries/portfolioQuery.js]: Why'd you do this?

**Review by moiseslorap — APPROVED** [review]

**Comment by johnbyrne727rit** [comment]
Are the tests not passing because of an expected exception? If so we can assert that the exception happens?

**Comment by JohnPMurray** [comment]
Nah the tests aren't passing because there should be an exception thrown in the code but there isn't

Edit: or its just the wrong exception being thrown like were getting a "can't read property x from null" instead of a "please enter a portfolio id"

File: backend/resolvers/queries/portfolioQuery.js (hunk 1 above)
Comment (JohnPMurray) [reply backend/resolvers/queries/portfolioQuery.js]: I guess I can go ahead and undo this

**Review by johnbyrne727rit — APPROVED** [review]
If the tests are failing because expected behavior isn't happening, I think its ok to get this in. It would be nice if we had more time to hunt whatever is happening down, but it's the last week of class :(

**Pull request merged**